### PR TITLE
Project mostly done. Ran into a few last minute questions.

### PR DESCRIPTION
**Pull request by brodriguez8774**
In attempted order of most important to least important:

1) You said that we're supposed to "use main to do the work and just use the interface to display." However, as I work on this, I'm finding very little opportunity to have main do the work. Instead, I'm unintentionally doing everything in the interface (mostly the case selection area). Obviously, it sounds like there's a better way to do this. Is it possible to create public methods from within main, and call them from my interface instead? Or is there another way to go about it?

2) Your instructions are to "Allow the user to load the wine list" which makes me think it should be an option in the menu, as opposed something done immediately on program load. However making it an option allows for the user to be dumb and attempt to manipulate/display the wine list before one is created. I don't know if you would prefer that I output an error message for this scenario (such as "please load a wine list first"), or if I should allow the user to add items to a blank list, search that manually created list, and print out "Wine list is empty" if nothing is currently available to print.

TLDR: How do you want user-error to be handled? IE, if the user DOES NOT load the wine list, and instead attempts to print, search, or add item to the list instead.

3) I have a bunch of "test" regions within my main. Would it probably be better to just create a testingClass and put them there or is it normal to keep such things in main?

### CODE REVIEW

**Comment by dbarneskvcc** [comment]
1. I was thinking that a UI object would be created in the Main. In fact, I was thinking that instances of all classes would be created in Main and passed to other classes as needed rather than 'newing' up an object inside one of the classes.  They could be passed via constructor or method.

With a UI object in Main, you would then call public functions from main to display  output and retrieve input. Within Main you would then 'decide' what to do, and call an appropriate public method on one of the other classes.

ie. You might call a ToString method on the wine collection that would return a string that needs to be output. You would then call some method on the UI class passing the returned output string to it.

The only exception I thought of to having the UI class handle all the I/O, is when a try / catch occurs. It would be good to have the UI handle all I/O, but it is okay, and may just be easier to output catch errors directly to the console where ever they happen. Exceptions in the catch aren't 'supposed' to happen anyway. That's why I think it is okay to do error output right there, and not struggle to get them back to the UI class. But that's your choice.

2. I was thinking that printing the list would be an option in the menu. It's okay if they try to print a blank list, but it might want to let them know that. In addition, I should be able to add items to a blank list and only print that. I could even load the CSV after I have added some of my own, and then tell it to print.

As far as error messages. Do whatever you think is necessary. It should be graceful, or find ways to prevent me from doing something boneheaded. If I can do something boneheaded, it should at least let me know that I did.

3. It appears as though your test region is simply testing out the other classes to ensure that they do what they are supposed to before you put them all together. In the real world, there would be a separate program that would do all of that work. The other program would be specifically designed to do testing and only testing. If we have time, I would like to show you all how to do it, but I'm not sure where we will be, or which assignment I want to do it in. For now, I think what you are doing is fine. I am assuming that when it is done you will either remove them, or leave the commented out, which is okay with me.

Hope that helps.
